dspinner/pin.go

```diff
@@ -981,15 +981,15 @@ func (p *pinner) rebuildIndexes(ctx context.Context) error {
 			indexer = p.cidRIndex
 			// Delete any direct index
 			err = p.cidDIndex.Delete(ctx, indexKey, pp.Id)
-			log.Infof("deleting stale pin index for cid %v", pp.Cid.String())
+			log.Errorf("deleting stale pin index for cid %v", pp.Cid.String())
```

Comment:
This appears to log the error message whether or not a stale pin was actually found.  Perhaps `Delete` needs to return a boolean, or there should be a find for the pin first.

Reply:
Let's not log deletions for now. I'll update the PR.